broker/query_plan_agg.go

```diff
@@ -109,8 +109,10 @@ func (mn *mergeNodeImpl) Execute(ctx context.Context) (result queryCom.AQLQueryR
 		}(i, c)
 	}
 
+	dataNodeWaitStart := utils.Now()
 	// TODO early merge before all results come back
 	wg.Wait()
+	utils.GetRootReporter().GetTimer(utils.TimeWaitedForDataNode).Record(utils.Now().Sub(dataNodeWaitStart))
```

Comment:
lint indent?

Reply:
it has the same indent as the above line.. must be github ui rendering bug